common/migrations/0073_merge_20160930_1934.py

```diff
@@ -0,0 +1,16 @@
+# -*- coding: utf-8 -*-
```

Comment:
What is this migration supposed to do? It doesn't seem like it will do anything.